roles/pulp-devel/templates/alias.bashrc.j2

```diff
@@ -66,6 +66,26 @@ pjournal() {
 _pjournal_help="Interact with the journal for pulp-related units
     pjournal takes optional journalctl args e.g. 'pjournal -r', runs pjournal -f by default"
 
+populate_pulp2() {
+    sudo systemctl stop httpd pulp_workers pulp_resource_manager pulp_celerybeat
+    mongo pulp_database --eval "db.dropDatabase()"
+    sudo rm -rf /var/lib/pulp/content
+    sudo rm -rf /var/lib/pulp/published
+    sudo -u apache pulp-manage-db;
+    sudo systemctl start httpd pulp_workers pulp_resource_manager pulp_celerybeat
+    pulp-admin login -u admin -p admin
+    pulp-admin iso repo create --feed https://repos.fedorapeople.org/pulp/pulp/fixtures/file/ --repo-id file
+    pulp-admin iso repo create --feed https://repos.fedorapeople.org/pulp/pulp/fixtures/file2/ --repo-id file2
```

Comment:
Let's sync  file2 and file-many  with on_demand policy, so we have various content in pulp 2.